avocado/utils/process.py

```diff
@@ -222,6 +222,7 @@ class CmdResult(object):
     :param stdout: String containing stdout of the process
     :param stderr: String containing stderr of the process
     :param duration: Elapsed wall clock time running the process
+    :param pid: ID of the process
```

Comment:
This is actually a documentation on the creation of `CmdResult` instance, that is, `CmdResult.__init__()`, so this parameter line is not appropriate here.